packages/react/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -365,10 +371,7 @@ const BreadcrumbsItem = React.forwardRef(({selected, className, ...rest}, ref) =
   return (
     <BoxWithFallback
       as="a"
-      className={clsx(className, classes.Item, {
-        [SELECTED_CLASS]: selected,
-        [classes.ItemSelected]: selected,
-      })}
+      className={clsx(className, classes.Item, selected && 'selected')}
```

Comment:
@pksjce I just moved all selected logic into `aria-current` in CSS, so we don't need that ItemSelected class. No idea what `selected` is doing, probably something in dotcom